### ISSUE TITLE
Try different learning rate schedulers for MART

### ISSUE THREAD

**Issue by tianshuocong** [IntelLabs/MART#71]
Hi! Thanks for the nice work!

Just FYI.

When I tried to add learning rate schedulers for MART (such as https://github.com/IntelLabs/MART/pull/26), I used `torch.optim.RMSprop` , and it returned
```
__init__() got an unexpected keyword argument 'maximize"
```

This bug comes from the inconsistent APIs of Pytorch. I updated PyTorch 1.12 to 1.13 and solved it.

**Comment by mzweilin** [comment]
@tianshuocong 
Thanks for reporting the issue. We have upgraded the dependency accordingly.
